vendor/assets/javascripts/ui-slider.js

```diff
@@ -0,0 +1,138 @@
+/*
+ jQuery UI Slider plugin wrapper
+*/
+angular.module('ui.slider', []).value('uiSliderConfig',{}).directive('uiSlider', ['uiSliderConfig', '$timeout', function(uiSliderConfig, $timeout) {
+    uiSliderConfig = uiSliderConfig || {};
+    return {
+        require: 'ngModel',
+        compile: function () {
+            return function (scope, elm, attrs, ngModel) {
+
+                function parseNumber(n, decimals) {
+                    return (decimals) ? parseFloat(n) : parseInt(n, 10);
+                }
+
+                var options = angular.extend(scope.$eval(attrs.uiSlider) || {}, uiSliderConfig);
+                // Object holding range values
+                var prevRangeValues = {
+                    min: null,
+                    max: null
+                };
+
+                // convenience properties
+                var properties = ['min', 'max', 'step'];
+                var useDecimals = (!angular.isUndefined(attrs.useDecimals)) ? true : false;
+
+                var init = function() {
+                    // When ngModel is assigned an array of values then range is expected to be true.
+                    // Warn user and change range to true else an error occurs when trying to drag handle
+                    if (angular.isArray(ngModel.$viewValue) && options.range !== true) {
+                        console.warn('Change your range option of ui-slider. When assigning ngModel an array of values then the range option should be set to true.');
+                        options.range = true;
+                    }
+
+                    // Ensure the convenience properties are passed as options if they're defined
+                    // This avoids init ordering issues where the slider's initial state (eg handle
+                    // position) is calculated using widget defaults
```

Comment:
Ahhh, I didn't notice that without reading the file's path. Could you add something in the header comment about where this is taken from?